plugins/modules/inventory_intent.py

```diff
@@ -2990,6 +3024,16 @@ def get_diff_merged(self, config):
 
         config['ip_address_list'] = devices_to_add
 
+        if self.config[0].get('update_mgmt_ipaddresslist'):
+            device_ip = self.config[0].get('update_mgmt_ipaddresslist')[0].get('existMgmtIpAddress')
+            is_device_exists = self.is_device_exist_in_ccc(device_ip)
+
+            if not is_device_exists:
+                self.status = "failed"
+                self.msg = "Device '{0}' not present in Cisco Catalyst Center so we cannot update the Management IP address".format(device_ip)
```

Comment:
Can we rephrase like below?
self.msg = "Unable to update the Management IP address because the device with IP '{0}' is not found in Cisco Catalyst Center.".format(device_ip)

Reply:
updated.